docs/_snippets/consolidated-query.mdx

```diff
@@ -0,0 +1,23 @@
+<ResponseField name="requestId" type="string" required>
+  The ID of the request.
```

Comment:
Maybe also mention that this is the same ID that's received as a result of `POST .../consolidated/query`?

Reply:
This is being used on both POST and GET unless i remove this and add to both

Reply:
The end result of the doc shouldn't be limited by how we structure our files. :)

I'll let you decide the best approach. 👍